awx_collection/plugins/modules/tower_schedule.py

```diff
@@ -0,0 +1,252 @@
+      required: False
+      type: str
+    name:
+      description:
+        - Name of this schedule.
+      required: True
+      type: str
+    new_name:
+      description:
+        - Setting this option will change the existing name (looked up via the name field.
+      required: False
+      type: str
+    description:
+      description:
+        - Optional description of this schedule.
+      required: False
+      type: str
+    extra_data:
+      description:
+        - Specify C(extra_vars) for the template.
+      required: False
+      type: dict
+      default: {}
+    inventory:
+      description:
+        - Inventory applied as a prompt, assuming job template prompts for inventory
+      required: False
+      type: str
+    scm_branch:
+      description:
+        - Branch to use in job run. Project default used if blank. Only allowed if project allow_override field is set to true.
+      required: False
+      type: str
+    job_type:
+      description:
+        - The job type to use for the job template.
+      required: False
+      type: str
+      choices:
+        - 'run'
+        - 'check'
+    job_tags:
+      description:
+        - Comma separated list of the tags to use for the job template.
+      required: False
+      type: str
+    skip_tags:
+      description:
+        - Comma separated list of the tags to skip for the job template.
+      required: False
+      type: str
+    limit:
+      description:
+        - A host pattern to further constrain the list of hosts managed or affected by the playbook
+      required: False
+      type: str
+    diff_mode:
+      description:
+        - Enable diff mode for the job template.
+      required: False
+      type: bool
+    verbosity:
+      description:
+        - Control the output level Ansible produces as the playbook runs. 0 - Normal, 1 - Verbose, 2 - More Verbose, 3 - Debug, 4 - Connection Debug.
+      required: False
+      type: int
+      choices:
+        - 0
+        - 1
+        - 2
+        - 3
+        - 4
+        - 5
+    unified_job_template:
+      description:
+        - Name of unified job template to schedule.
+      required: False
+      type: str
+    enabled:
+      description:
+        - Enables processing of this schedule.
+      required: False
+      type: bool
+    state:
+      description:
+        - Desired state of the resource.
+      choices: ["present", "absent"]
+      default: "present"
+      type: str
+    tower_oauthtoken:
+      description:
+        - The Tower OAuth token to use.
+      required: False
+      type: str
+      version_added: "3.7"
+extends_documentation_fragment: awx.awx.auth
+'''
+
+EXAMPLES = '''
+- name: Build a schedule for Demo Job Template
+  tower_schedule:
+    name: "{{ sched1 }}"
+    state: present
+    unified_job_template: "Demo Job Template"
+    rrule: "DTSTART:20191219T130551Z RRULE:FREQ=WEEKLY;INTERVAL=1;COUNT=1"
+  register: result
+  ignore_errors: True
```

Comment:
why do these examples have `ignore_errors`? Deleting that might be best for clarity

Reply:
Good catch, I think I was playing around with extracting the example docs out of our unit testing and those were in there. Should be removed from both now.